testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-oauth2.yaml

```diff
@@ -0,0 +1,14 @@
+databases:
```

Comment:
This file should be placed in product tests launcher and something like below should be used:
```
        String temptoConfig = "/docker/product-tests/conf/tempto/tempto-configuration-for-oauth2.yaml";
        builder.configureContainer(TESTS, dockerContainer -> dockerContainer
                .withEnv("TEMPTO_CONFIG_FILES", temptoConfigFiles ->
                        temptoConfigFiles
                                .map(files -> files + "," + temptoConfig)
                                .orElse(temptoConfig))
                .withCopyFileToContainer(forHostPath(comfig.getPath("tempto-configuration.yaml")), temptoConfig)
                ```